content/zmc-challenges/first-website/your-first-html-page/_index.md

```diff
@@ -34,61 +34,61 @@ You can also experiment a bit by writing HTML code and seeing what it does. Expe
 
 Your mission is to make a website of your very own. It will just be one page for now. 
 
-It's very important that you follow the instructions exactly, in the next step you will be deploying your website to the web.  It will need to do exactly the right thing!
+It's essential that you follow the instructions exactly, in the next step you will be deploying your website to the web.  It will need to do exactly the right thing!
 
-### 1. Create a new html file called `index.html`
+### 1. Create a new HTML file called `index.html`
 
-It is important to name the file exactly right! We'll be putting it on the internet in the next step. 
+It is important to name the file exactly right! We'll be putting it on the Internet in the next step. 
 
 If you want to you can just edit the html file we made in the last step. So long as the file name is correct then we are good. 
 
 Please note that `index.html` is NOT the same as `Index.html`. Capital letters matter a lot in code. Please be extra careful about naming your files with the right case!
 
 ### 2. Let's make some content 
 
-You'll be adding a bunch of content to your html file. Please make very sure that you are meeting all the requirements below:
+You'll be adding a bunch of content to your html file. Please make sure that you are meeting all the requirements below:
 
 ### 2.1 Add a `<head>` element
```

Comment:
This heading should be a level 4 since it falls under `2. Let's make some content `